images/dind/preentry.sh

```diff
@@ -6,6 +6,15 @@ if [ $fsdriver == "overlay" ]; then
   fsdriver="overlay2"
 fi
 
+mkdir -p /etc/docker
+/bin/cat > /etc/docker/daemon.json <<EOF
```

Comment:
similar to `--storage-driver`, docker has an option for this one we could pass below: `dockerd --help | grep mtu`